arch/x86/64/source/ArchMemory.cpp

```diff
@@ -35,7 +33,7 @@ bool ArchMemory::checkAndRemove(pointer map_ptr, uint64 index)
   return true;
 }
 
-bool ArchMemory::unmapPage(uint64 virtual_page)
+void ArchMemory::unmapPage(uint64 virtual_page)
```

Comment:
not sure this is a good idea? students commonly change map/unmapPage in a way that the boolean is used on the outside to indicate success or whether some clean up is necessary?